src/rsp_restspawner/spawner.py

```diff
@@ -100,7 +90,7 @@ async def poll(self) -> Optional[int]:
         client = await get_client()
         r = await client.get(
             f"{self.ctrl_url}/labs/{self.user.name}",
-            headers=await self._configure_client_headers(),
```

Comment:
Huh.  Surprised this worked with user auth.

Reply:
Yeah, I'm not sure how that had worked. When I started testing with mobu, it failed, and it seems to require admin auth. The whole auth situation is indeed a mess as you pointed out earlier, and it's on my list to overhaul that, probably by providing some duplicate routes similar to `user-status` so that the whole thing is a bit more coherent.